modules/ROOT/pages/dataweave-formats-avro.adoc

```diff
@@ -99,36 +103,38 @@ output application/avro schemaUrl="classpath://schema.json"
 ----
 
 
-// CONFIG PROPS ///////////////////////////////////////////////////////
-
 [[properties]]
 == Configuration Properties
 
-DataWeave supports the following configuration properties for Avro.
+DataWeave supports the following configuration properties for the Avro format.
 
-=== Reader Properties (for Avro)
+
+[[reader_properties]]
+=== Reader Properties
 
 The Avro format accepts properties that provide instructions for reading input data.
 
+
 [cols="1,1,1,3a", options="header"]
 |===
-| Parameter | Type | Default | Description
-| `schemaUrl`   (Required) | `String` | `null` | The URL for the Avro schema. Valid values are `classpath://`, `file://`, or `http://`.
+|Parameter |Type |Default|Description
+|`schemaUrl`  (Required) |`String`|`null`|The URL for the `Avro` schema. Valid values are `classpath://`, `file://`, or `http://`.
 |===
 
-=== Writer Properties (for Avro)
+[[writer_properties]]
+=== Writer Properties
```

Comment:
TODO: "THE <format_name> format" --> "This format" everywhere for same reasons as above (applies to reader and writer property intros)